File: students/students/src/MON.Models/Institution/StudentInfoModel.cs
using MON.Models.StudentModels;

namespace MON.Models.Institution
{
    public class StudentInfoModel : StudentSummaryBaseModel
    {
        public string BirthPlace { get; set; }
        public string Phone { get; set; }
        public bool? HasSpecialNeeds { get; set; }
        public string Usernames { get; set; }
        public string InitialPasswords { get; set; }
        public int ClassId { get; set; }
        public int? ClassNumber { get; set; }
        public string ClassName { get; set; }
        public string EduFormName { get; set; }
        public string MainClassName { get; set; }
    }
}

File: students/students/src/MON.SPA/src/models/institution/studentInfoModel.js
export class StudentInfoModel{
    constructor(obj = {}){
        this.personId = obj.personId || null;
        this.fullName = obj.fullName || '';
        this.classNumber = obj.classNumber || null;
        this.birthPlace = obj.birthPlace || '';
        this.phone = obj.phone || '';
        this.hasSpecialNeeds = obj.hasSpecialNeeds || false;
        this.eduFormName = obj.eduFormName || '';
        this.username = obj.username,
        this.initialPassword = obj.initialPassword;
    }
}